test/parallel/test-cluster-shared-handle-bind-reuseaddr.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+var common = require('../common');
+var assert = require('assert');
+var cluster = require('cluster');
+var dgram = require('dgram');
+var spawn = require('child_process').spawn;
+
+if (common.isWindows) {
+  console.log('1..0 # Skipped: `reuseAddr: true` not supported on Windows');
+  return;
+}
+
+if (cluster.isMaster) {
+  cluster.fork().on('exit', common.mustCall(function(exitCode) {
+    assert.strictEqual(exitCode, 0);
+  }));
+} else {
+  var isExtraChild = (process.argv.length === 3);
+  var port = (isExtraChild ? +process.argv[2] : 0);
+  var socket = dgram.createSocket({type: 'udp4', reuseAddr: true});
+  socket.bind(port, 'localhost', common.mustCall(function() {
+    if (isExtraChild)
+      return process.exit(0);
+    port = socket.address().port;
+    var child = spawn(process.execPath, [__filename, port]);
+    child.on('close', function(code, signal) {
+      assert.strictEqual(code, 0);
+      assert.strictEqual(signal, null);
+      process.exit(0);
```

Comment:
Ok so apparently loading `cluster` in the extra child process was causing problems...